roles/confluent.control_center/tasks/main.yml

```diff
@@ -3,6 +3,15 @@
     name: confluent.common
   when: not common_role_completed|bool
 
+- name: Gather OS Facts
+  setup:
+    filter: "{{item}}"
+    gather_subset:
```

Comment:
Might be worth while to add a comment stating this ignores all facts, and only gathers what is specified in the filter.